hash_table_implementation/hash_table.py

```diff
@@ -0,0 +1,86 @@
+class HashTable(object):
+
+    def __init__(self, size):
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpd7804up6/hash_table_implementation/hash_table.py
+++ b/tmp/tmpd7804up6/hash_table_implementation/hash_table.py
@@ -34,6 +34,7 @@
                 # Otherwise replace old value
                 else:
                     self.data[nslt] = data
+
     @classmethod
     def hashfunction(self, key, size):
         # Remainder Method
```